packages/storybook-webpack-angular-types-plugin/src/lib/webpack-angular-types-plugin/plugin.ts

```diff
@@ -172,6 +144,48 @@ export class WebpackAngularTypesPlugin {
 		return groupedExportsInformation;
 	}
 
+	private addToGroupedExportsInformation(
+		exportsInformationMap: Map<string, ExportsInformation>,
+		{
+			functionInformation,
+			constantInformation,
+		}:
+			| { functionInformation: FunctionInformation; constantInformation?: never }
+			| { constantInformation: ConstantInformation; functionInformation?: never },
+	): Map<string, ExportsInformation> {
+		const newExportInformation = functionInformation ?? constantInformation;
+
+		const name = newExportInformation.name;
+
+		const entryKey = functionInformation ? 'functionsInformation' : 'constantsInformation';
+
+		exportsInformationMap.set(name, {
+			functionsInformation: [],
+			constantsInformation: [],
+			[entryKey]: [newExportInformation],
+		});
```

Comment:
Still not sure what purpose this serves. Do you read from this map by function/constant name somewhere?

Reply:
Yes. In line 228 ;)